commit 578cd59dd87d0c3f8c507a071b5a242e75cc25f9
Author: Sage Weil <sweil@redhat.com>
Date:   Mon May 22 18:47:40 2017 -0400

    mgr: drop useless __func__ prints
    
    This is part of the default prefix.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/Mgr.cc b/src/mgr/Mgr.cc
index cbc443f3a32..dc4daed1b59 100644
--- a/src/mgr/Mgr.cc
+++ b/src/mgr/Mgr.cc
@@ -585,6 +585,6 @@ void Mgr::handle_mgr_digest(MMgrDigest* m)
 
 void Mgr::tick()
 {
-  dout(0) << __func__ << dendl;
+  dout(0) << dendl;
   server.send_report();
 }
